src/open_inwoner/accounts/tests/test_auth.py

```diff
@@ -962,18 +966,27 @@ def test_eherkenning_user_is_redirected_to_necessary_registration(
 
         user = eHerkenningUserFactory.create(kvk="12345678", email="example@localhost")
 
-        response = self.app.get(reverse("pages-root"), user=user)
+        self.client.force_login(
+            user, backend=EHerkenningSessionContext._expected_auth_backends()[0]
+        )
+        response = self.client.get(reverse("pages-root"), user=user)
```

Comment:
Do we need to switch from `app` to `client` for this test (we're using the `WebTest`'s `app` in the other tests)?